micromark/exports-html.mjs

```diff
@@ -2,7 +2,7 @@
 
 /* eslint-disable max-len, n/no-missing-import */
 
-export { compile } from "micromark/lib/compile";
+export { compile } from "micromark/lib/compile.js";
```

Comment:
I assume all these changes here will go away when you rebase onto the latest changes in `next`?